content/blog/2019-10-02-dev-guide-update-fall19.md

```diff
@@ -0,0 +1,94 @@
+
+
+
+### Updates to our policies and guidance
+
+#### Scope
+
+We've introduced an important change for anyone thinking of submitting a package. The former "reproducibility" category has been split up into four new categories (with "reproducibility" itself removed). They are:
+
+- data validation and testing
+- workflow automation
+- version control
+- citation management and bibliometrics
+
+For discussion and changes see [ropensci/software-review-meta#81](https://github.com/ropensci/software-review-meta/issues/81) and [ropensci/dev_guide#184](https://github.com/ropensci/dev_guide/pull/184).
+
+#### Documentation
+
+rOpenSci's [Jeroen Ooms](https://ropensci.org/authors/jeroen-ooms/) has built out a new way for rOpenSci packages to improve documentation for their packages. Many of you have likely been building package documentation websites with [pkgdown][]. There were ways to automate the building of these documentation pages - but nothing as easy as we have now. Docs for each rOpenSci package are now built automatically and can be found at `https://docs.ropensci.org/{package}`, where `{package}` is the package name, e.g., the page for the [bowerbird][] package is <https://docs.ropensci.org/bowerbird/>.
+
+We've changed our guidelines regarding README's and vignettes. If you have chunks shared between the README and any vignettes, then [we recommend using Rmarkdown chunks](https://devguide.ropensci.org/building.html#documentation). Related issues: [ropensci/dev_guide#159](https://github.com/ropensci/dev_guide/issues/159) and [ropensci/dev_guide#161](https://github.com/ropensci/dev_guide/issues/161).
+
+Last, we've removed our recommendation about using the `roxygen2` development version because the latest stable version has what is needed regarding tags and the `rev` contributor role. (thanks [Hugo Gruson][hugog] - [ropensci/dev_guide#165](https://github.com/ropensci/dev_guide/issues/165))
+
+#### CRAN checks
+
+We've added [explanations of CRAN checks](https://devguide.ropensci.org/building.html#cranchecks) to the packaging section, including that they can use the CRAN checks API, in particular see `rodev::use_cchecks_badge()` for adding a badge to your README to let users know the status of your CRAN checks.
+
+#### R helpers
+
+The [rodev][] package is a new helper package, maintained by Maëlle. The goal of rodev is to help rOpenSci package developers with common tasks, and to promote best practices. We've added mentions of rodev functions throughout the guide where appropriate.
+
+Like rodev, [usethis][] is a helper package for maintainers. We've also added to the guide recommendations for usethis functions to use for adding testing or vignette infrastructure.
```

Comment:
A bit misleading since rodev is built on usethis so maybe invert paragraph order.

Reply:
done